store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/utils/AccumuloStoreConstants.java

```diff
@@ -45,6 +52,12 @@ public final class AccumuloStoreConstants {
      */
     public static final long DEFAULT_TIMESTAMP = 1L;
 
+    // Time sensitive aggregation functions where an actual timestamp must applied to the key
```

Comment:
Switched to class comparison but was reluctant to enforce sub type just in case but you are right in practise usually always going to be binary operators.